pkg/util/escape.go

```diff
@@ -20,6 +20,19 @@ import (
 	"strings"
 )
 
+// EscapeQualifiedPluginName converts a plugin name in the format
+// vendor/pluginname into a proper ondisk vendor~pluginname plugin directory
+// format.
+func EscapePluginName(in string) string {
+	return strings.Replace(in, "/", "~", -1)
+}
+
+// EscapeQualifiedPluginName converts a plugin directory name in the format
```

Comment:
Looks like you missed the `EscapeQualifiedNameForDisk` comment above. Could you address it please.